extensions/awell/v1/actions/isPatientEnrolledInCareFlow/config/fields.ts

```diff
@@ -33,6 +33,14 @@ export const fields = {
     type: FieldType.STRING,
     required: false,
   },
+  dayRange: {
+    id: 'dayRange',
+    label: 'Day range',
+    description:
+      'Specify the day range to look for the care flows that were started in the past. e.g last 30 days.',
```

Comment:
I'd make it more verbose that what they need to enter is a number that represents the amount of days to look back.


```suggestion
    label: 'Past day range',
    description:
      'The number of days to look back for care flows that were started in the past, e.g last 30 days.',
```